drivers/nash.js

```diff
@@ -0,0 +1,33 @@
+const Driver = require('../models/driver');
+const request = require('../lib/request');
+const Ticker = require('../models/ticker');
+const { parseToFloat } = require('../lib/utils.js');
+
+/**
+ * @memberof Driver
+ * @augments Driver
+ */
+class Nash extends Driver {
+  /**
+   * @augments Driver.fetchTickers
+   * @returns {Promise.Array<Ticker>} Returns a promise of an array with tickers.
+   */
+  async fetchTickers() {
+    const markets = await request('https://app.nash.io/api/cmc/ticker');
```

Comment:
Mh, this is kind of a weird endpoint they've made! All the pair are reversed, even the prices haha!
After searching the internet I came across this one; its a GraphQL query that we could probably use:

```suggestion
    const { data: { listTickers } } = await request({
      method: 'POST',
      url: 'https://app.nash.io/api/graphql',
      headers: { Accept: 'application/json' },
      body: '{"operationName":"Tickers","variables":{},"query":"query Tickers {\n  listTickers {\n    ...tickerFields\n    __typename\n  }\n}\n\nfragment tickerFields on Ticker {\n  id\n  market {\n    ...marketFields\n    __typename\n  }\n  priceChange_24hPct\n  highPrice_24h {\n    ...currencyPriceFields\n    __typename\n  }\n  lowPrice_24h {\n    ...currencyPriceFields\n    __typename\n  }\n  volume_24h {\n    ...currencyAmountFields\n    __typename\n  }\n  lastPrice {\n    ...currencyPriceFields\n    __typename\n  }\n  usdLastPrice {\n    ...currencyPriceFields\n    __typename\n  }\n  usdLastPriceB {\n    ...currencyPriceFields\n    __typename\n  }\n  bestBidPrice {\n    ...currencyPriceFields\n    __typename\n  }\n  bestAskPrice {\n    ...currencyPriceFields\n    __typename\n  }\n  __typename\n}\n\nfragment currencyAmountFields on CurrencyAmount {\n  amount\n  currency\n  __typename\n}\n\nfragment currencyPriceFields on CurrencyPrice {\n  amount\n  currencyA\n  currencyB\n  __typename\n}\n\nfragment marketFields on Market {\n  aAsset {\n    ...assetFields\n    __typename\n  }\n  aUnit\n  aUnitPrecision\n  bAsset {\n    ...assetFields\n    __typename\n  }\n  bUnit\n  bUnitPrecision\n  id\n  minTickSize\n  minTradeSize\n  minTradeSizeB\n  minTradeIncrement\n  minTradeIncrementB\n  name\n  status\n  priceGranularity\n  primary\n  __typename\n}\n\nfragment assetFields on Asset {\n  blockchain\n  blockchainPrecision\n  depositPrecision\n  hash\n  name\n  symbol\n  withdrawalPrecision\n  __typename\n}\n"}',
    });
```